ecpy_hqc_legacy/tasks/tasks/instr/pna_tasks.py

```diff
@@ -377,7 +377,7 @@ def perform(self):
 
         waiting_time = self.channel_driver.sweep_time
         self.driver.fire_trigger(self.channel)
-        time.sleep(waiting_time)
+        time.sleep(1.5*waiting_time)
```

Comment:
Why this change ?

Reply:
If I keep this as before, often the PNA doesn't have time to take its trace, and the data looks like the beginning of a trace, and then 0s. I guess that the instrument is not capable of knowing exactly what its sweep time is, since there must be a time overhead to set various parameters before or after the sweep.

Reply:
This means that the ```check_operation_complete``` method is not doing its job properly in this case as it should block further execution. What bother me most with this change is that for long sweeps we may end up wasting a lot of time.

Reply:
Ok fine I put it back. The fact is this doesn't work, so it needs to be changed locally. maybe it can be a parameter in the task which the user can set.